FuzzedSock compiler warnings when building current master

```
  CXX      test/fuzz/fuzz-addition_overflow.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/addition_overflow.cpp:7:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/addition_overflow.cpp:7:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-addrdb.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/addrdb.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/addrdb.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-addrman.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/addrman.cpp:11:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/addrman.cpp:11:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-asmap.o
  CXX      test/fuzz/fuzz-asmap_direct.o
  CXX      test/fuzz/fuzz-autofile.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/autofile.cpp:9:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/autofile.cpp:9:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-banman.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/banman.cpp:10:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/banman.cpp:10:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-base_encode_decode.o
  CXX      test/fuzz/fuzz-bech32.o
  CXX      test/fuzz/fuzz-block.o
  CXX      test/fuzz/fuzz-block_header.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/block_header.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/block_header.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-blockfilter.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/blockfilter.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/blockfilter.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-bloom_filter.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/bloom_filter.cpp:9:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/bloom_filter.cpp:9:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-buffered_file.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/buffered_file.cpp:9:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/buffered_file.cpp:9:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-chain.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/chain.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/chain.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-checkqueue.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/checkqueue.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/checkqueue.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-coins_view.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/coins_view.cpp:18:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/coins_view.cpp:18:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-connman.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from test/fuzz/connman.cpp:7:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/connman.cpp:12:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-crypto.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/crypto.cpp:15:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/crypto.cpp:15:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-crypto_aes256.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/crypto_aes256.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/crypto_aes256.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-crypto_aes256cbc.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/crypto_aes256cbc.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/crypto_aes256cbc.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-crypto_chacha20.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/crypto_chacha20.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/crypto_chacha20.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-crypto_chacha20_poly1305_aead.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/crypto_chacha20_poly1305_aead.cpp:9:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/crypto_chacha20_poly1305_aead.cpp:9:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-crypto_common.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/crypto_common.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/crypto_common.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-crypto_hkdf_hmac_sha256_l32.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/crypto_hkdf_hmac_sha256_l32.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/crypto_hkdf_hmac_sha256_l32.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-crypto_poly1305.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/crypto_poly1305.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/crypto_poly1305.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-cuckoocache.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/cuckoocache.cpp:9:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/cuckoocache.cpp:9:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-data_stream.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from test/fuzz/data_stream.cpp:6:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/data_stream.cpp:9:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-decode_tx.o
  CXX      test/fuzz/fuzz-descriptor_parse.o
  CXX      test/fuzz/fuzz-deserialize.o
  CXX      test/fuzz/fuzz-eval_script.o
  CXX      test/fuzz/fuzz-fee_rate.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/fee_rate.cpp:9:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/fee_rate.cpp:9:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-fees.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/fees.cpp:9:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/fees.cpp:9:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-flatfile.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/flatfile.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/flatfile.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-float.o
  CXX      test/fuzz/fuzz-golomb_rice.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/golomb_rice.cpp:10:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/golomb_rice.cpp:10:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-hex.o
  CXX      test/fuzz/fuzz-http_request.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/http_request.cpp:9:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/http_request.cpp:9:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-integer.o
In file included from ./netbase.h:15,
                 from test/fuzz/integer.cpp:14:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/integer.cpp:26:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-key.o
  CXX      test/fuzz/fuzz-key_io.o
  CXX      test/fuzz/fuzz-kitchen_sink.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/kitchen_sink.cpp:10:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/kitchen_sink.cpp:10:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-load_external_block_file.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/load_external_block_file.cpp:9:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/load_external_block_file.cpp:9:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-locale.o
  CXX      test/fuzz/fuzz-merkleblock.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/merkleblock.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/merkleblock.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-message.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/message.cpp:9:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/message.cpp:9:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-muhash.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/muhash.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/muhash.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-multiplication_overflow.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/multiplication_overflow.cpp:7:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/multiplication_overflow.cpp:7:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-net.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from test/fuzz/net.cpp:7:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/net.cpp:15:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-net_permissions.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/net_permissions.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/net_permissions.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-netaddress.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/netaddress.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/netaddress.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-node_eviction.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from test/fuzz/node_eviction.cpp:5:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/node_eviction.cpp:10:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-p2p_transport_deserializer.o
  CXX      test/fuzz/fuzz-parse_hd_keypath.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/parse_hd_keypath.cpp:7:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/parse_hd_keypath.cpp:7:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-parse_iso8601.o
  CXX      test/fuzz/fuzz-parse_numbers.o
  CXX      test/fuzz/fuzz-parse_script.o
  CXX      test/fuzz/fuzz-parse_univalue.o
  CXX      test/fuzz/fuzz-policy_estimator.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/policy_estimator.cpp:9:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/policy_estimator.cpp:9:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-policy_estimator_io.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/policy_estimator_io.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/policy_estimator_io.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-pow.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/pow.cpp:11:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/pow.cpp:11:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-prevector.o
  CXX      test/fuzz/fuzz-primitives_transaction.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/primitives_transaction.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/primitives_transaction.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-process_message.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from test/fuzz/process_message.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/process_message.cpp:16:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-process_messages.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from test/fuzz/process_messages.cpp:6:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/process_messages.cpp:11:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-protocol.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/protocol.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/protocol.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-psbt.o
  CXX      test/fuzz/fuzz-random.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/random.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/random.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-rbf.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/rbf.cpp:10:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/rbf.cpp:10:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-rolling_bloom_filter.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/rolling_bloom_filter.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/rolling_bloom_filter.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-script.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/script.cpp:21:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/script.cpp:21:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-script_assets_test_minimizer.o
  CXX      test/fuzz/fuzz-script_bitcoin_consensus.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/script_bitcoin_consensus.cpp:9:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/script_bitcoin_consensus.cpp:9:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-script_descriptor_cache.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/script_descriptor_cache.cpp:10:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/script_descriptor_cache.cpp:10:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-script_flags.o
  CXX      test/fuzz/fuzz-script_interpreter.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/script_interpreter.cpp:9:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/script_interpreter.cpp:9:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-script_ops.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/script_ops.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/script_ops.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-script_sigcache.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/script_sigcache.cpp:12:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/script_sigcache.cpp:12:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-script_sign.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/script_sign.cpp:15:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/script_sign.cpp:15:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-scriptnum_ops.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/scriptnum_ops.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/scriptnum_ops.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-secp256k1_ec_seckey_import_export_der.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/secp256k1_ec_seckey_import_export_der.cpp:9:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/secp256k1_ec_seckey_import_export_der.cpp:9:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-secp256k1_ecdsa_signature_parse_der_lax.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/secp256k1_ecdsa_signature_parse_der_lax.cpp:9:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/secp256k1_ecdsa_signature_parse_der_lax.cpp:9:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-signature_checker.o
  CXX      test/fuzz/fuzz-signet.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/signet.cpp:12:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/signet.cpp:12:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-socks5.o
In file included from ./netbase.h:15,
                 from test/fuzz/socks5.cpp:5:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/socks5.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-span.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/span.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/span.cpp:8:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-spanparsing.o
  CXX      test/fuzz/fuzz-string.o
In file included from ./netbase.h:15,
                 from test/fuzz/string.cpp:8:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/string.cpp:20:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-strprintf.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/strprintf.cpp:7:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/strprintf.cpp:7:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-system.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/system.cpp:7:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/system.cpp:7:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-timedata.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/timedata.cpp:7:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/timedata.cpp:7:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-torcontrol.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/torcontrol.cpp:7:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/torcontrol.cpp:7:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  CXX      test/fuzz/fuzz-transaction.o
  CXX      test/fuzz/fuzz-tx_in.o
  CXX      test/fuzz/fuzz-tx_out.o
  CXX      test/fuzz/fuzz-txrequest.o
  CXX      test/fuzz/libtest_fuzz_a-fuzz.o
  CXX      test/fuzz/libtest_fuzz_a-util.o
In file included from ./i2p.h:13,
                 from ./net.h:17,
                 from ./test/fuzz/util.h:16,
                 from test/fuzz/util.cpp:5:
./util/sock.h:61:19: warning: ‘virtual Sock& Sock::operator=(Sock&&)’ was hidden [-Woverloaded-virtual]
   61 |     virtual Sock& operator=(Sock&& other);
      |                   ^~~~~~~~
In file included from test/fuzz/util.cpp:5:
./test/fuzz/util.h:533:7: warning:   by ‘FuzzedSock& FuzzedSock::operator=(const FuzzedSock&)’ [-Woverloaded-virtual]
  533 | class FuzzedSock : public Sock
      |       ^~~~~~~~~~
  AR       libtest_fuzz.a
```
```
$ gcc --version
gcc (Gentoo 9.3.0-r1 p3) 9.3.0
Copyright (C) 2019 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```
```
$ git log | head -n 1
commit ed25cb58f605ba583c735f330482df0bf9348f3a
```

--- a/src/test/fuzz/util.h
+++ b/src/test/fuzz/util.h
@@ -543,6 +543,12 @@ public:
     {
     }
 
+    FuzzedSock& operator=(Sock&& other) override
+    {
+        assert(false && "Not implemented yet.");
+        return *this;
+    }
+
     SOCKET Get() const override
     {
         assert(false && "Not implemented yet.");

